packages/plugins/rubygem-angular-rails-templates/rubygem-angular-rails-templates.spec

```diff
@@ -1,80 +1,65 @@
-# Template for Rubygem's spec file
-# It should work with f18, f19, rhel6 and rhel6 with SCL
-
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name angular-rails-templates
 
-Name:      %{?scl_prefix}rubygem-%{gem_name}
-Version:   1.1.0
-Release:   2%{?dist}
-Epoch:     1
```

Comment:
You lost the epoch when regenerating

Reply:
Now I'm wondering why 2db9adc930e7bb5a6fc802df0ff3933e410205b3 added an epoch in the first place.

Reply:
I am not sure I wanna know

Reply:
https://github.com/theforeman/foreman-packaging/pull/532 actually explains it rather nicely.